scripts/jenkins/jobs-obs/openstack-cleanvm.yaml

```diff
@@ -10,6 +10,7 @@
             - SLE_12-64-minimal
             - SLE_12_SP1-64-minimal
             - openSUSE-13.2-64-minimal
+            - Leap-42.1-64-minimal
```

Comment:
The real name is openSUSE Leap 42.1, I would like to maintain the openSUSE prefix here